react/features/prejoin/actions.js

```diff
@@ -311,8 +311,11 @@ export function replaceVideoTrackById(deviceId: Object) {
             const tracks = getState()['features/base/tracks'];
             const newTrack = await createLocalTrack('video', deviceId);
             const oldTrack = getLocalVideoTrack(tracks)?.jitsiTrack;
+            const localTrack = getLocalVideoTrack(getState()['features/base/tracks']);
 
-            dispatch(replaceLocalTrack(oldTrack, newTrack));
+            dispatch(
```

Comment:
Instead of doing this, we should check if video is muted in base/media, which signifies user intent. And we should do this outside of this function, on the caller side.